glean_parser/util.py

```diff
@@ -346,29 +365,24 @@ def is_expired(expires: str) -> bool:
     elif expires == "expired":
         return True
     else:
-        try:
-            if sys.version_info < (3, 7):
-                date = iso8601.parse_date(expires).date()
-            else:
-                date = datetime.date.fromisoformat(expires)
-        except ValueError:
-            raise ValueError(
-                f"Invalid expiration date '{expires}'. "
-                "Must be of the form yyyy-mm-dd in UTC."
-            )
+        date = parse_expires(expires)
         return date <= datetime.datetime.utcnow().date()
 
 
 def validate_expires(expires: str) -> None:
     """
-    Raises ValueError if `expires` is not valid.
+    Raises a ValueError in case the date is more than 730 days (~2 years) in the future
```

Comment:
or if `expires` isn't ISO8601 parseable.

Reply:
Yeap, I had that there, but I removed it because we should never get this errors since it is validated by the schema. Since we _could_ throw, according tot he code, I'll add it back.